src/sidebar/components/menu.js

```diff
@@ -21,6 +32,80 @@ const menuArrow = className => (
  */
 let ignoreNextClick = false;
 
+function isElementVisible(element) {
+  return element.offsetParent !== null;
+}
+
+function MenuList({ align, menuButtonId, contentClass, closeMenu, children }) {
```

Comment:
I'm thinking here that if we split out a component that wraps the _content_ of a menu, then we can let it handle the keyboard navigation when an item in that menu is focused. We can then re-use this in `GroupListItem` to wrap the submenu.